docs/content/configuration/prefill-disk-cache.md

```diff
@@ -0,0 +1,87 @@
+
+This document will guide you through packaging up a single module called `github.com/my/module`, and inserting it into the Athens disk storage.
+
+# First, get the tools
+
+You'll need to produce the following assets from module source code:
+
+- `source.zip` - just the Go source code, packaged in a zip file
+- `go.mod` - just the `go.mod` file from the module
+- `$VERSION.info` - metadata about the module
+
+The `source.zip` file has a specific directory structure and the `$VERSION.info` has a JSON structure, both of which you'll need to get right in order for Athens to serve up the right dependency formats that the Go toolchain will accept.
+
+>We don't recommend that you create these assets yourself. Instead, use [pacmod](https://github.com/plexsystems/pacmod)
+
+To install the `pacmod` tool, run `go install` like this:
+
+```console
+$ go install github.com/plexsystems/pacmod
+```
+
+This command will install the `pacmod` binary to your `$GOPATH/bin/pacmod` directory, so make sure that is in your `$PATH`.
+
+# Next, run `pacmod` to create assets
+
+After you have `pacmod`, you'll need the module source code that you want to package. Before you run the command, set the `VERSION` variable in your environment to the version of the module you want to generate assets for.
+
+Below is an example for how to configure it.
+
+```console
+$ export VERSION="v1.0.0
+```
+
+>Note: make sure your `VERSION` variable starts with a `v`
+
+Next, navigate to the top-level directory of the module source code, and run `pacmod` like this:
+
+```console
+$ pacmod pack $VERSION .
+```
+
+Once this command is done, you'll notice three new files in the same same directory you ran the command from:
+
+- `go.mod`
+- `$VERSION.info`
+- `$VERSION.zip`
+
+# Next, move assets into Athens storage directory
+
+Now that you have assets built, you need to move them into the location of the Athens disk storage. In the below commands, we'll assume `$STORAGE_ROOT` is the environment variable that points to the top-level directory that Athens uses for its on-disk.
+
+>If you set up Athens with the `$ATHENS_DISK_STORAGE_ROOT` environment variable, the root of this storage location is the value of this environment variable. Use `export STORAGE_ROOT=$ATHENS_DISK_STORAGE_ROOT` to prepare your environment for the below commands.
+
+First create the subdirectory into which you'll move the assets you created:
+
+```console
+$ mkdir -p $STORAGE_ROOT/github.com/my/module/$VERSION
+```
+
+Finally, make sure that you're still in the module source repository root directory (the same as you were in when you ran the `pacmod` command), and move your three new files into the new directory you just created:
+
+```console
+$ mv go.mod $STORAGE_ROOT/github.com/my/module/$VERSION/go.mod
+$ mv $VERSION.info $STORAGE_ROOT/github.com/my/module/$VERSION/$VERSION.info
+$ mv $VERSION.zip $STORAGE_ROOT/github.com/my/module/$VERSION/source.zip
```

Comment:
The `.zip` file for modules is the same name as the `.info` file ( $VERSION )

Reply:
@jpreese I think what you said went over my head. What should I change here?

Reply:
Nothing! I think we're good here. There was a lot of confusion around how Athens stores its `.zip` by naming it `source.zip` and how the Go modules toolchain stores its zips (`$VERSION.zip`). Though from what I recall Athens still handles it correctly. I have a comment around here somewhere when I dove into this a bit ago.. but I've forgotten 🤣

Reply:
ohhhhh, I got you 😄 

Thanks for clarifying!